Source/CustomMapInitData.cs

```diff
@@ -1,234 +0,0 @@
-﻿using RimWorld;
```

Comment:
This file seemed to be dead code that was using non-existent types such as StandardEquipment.